dashboards/cluster.jsonnet

```diff
@@ -19,7 +19,20 @@ local userNodes = graphPanel.new(
   min=0,
 ).addTarget(
   prometheus.target(
-    expr='sum(kube_node_labels) by (label_cloud_google_com_gke_nodepool)',
+    |||
+      # sum up all nodes by nodepool
+      sum(
+        # Get a list of all the nodes and which pool they are in. Group 
+        # aggregator is used because we only expect each node to exist
+        # in a single pool. Unfortunately, if a node pool is rotated it may 
```

Comment:
Ah, I think the explanation is slightly different. `kube_pod_labels` comes from https://github.com/kubernetes/kube-state-metrics, and there is a particular label (`kubernetes_node`) that lists the node on which the `kube-state-metrics` pod is running! So that's totally irrelevant to these queries, but when a nodepool is rotated it caused there to exist two metrics with the same `node` value (which we care about) but different `kubernetes_node` values (because kube-state-metrics was running in a different node, even though we don't care about that). So I think this `group` really just drops all labels except the two we care about to avoid messing things up.